src/app/app.component.ts

```diff
@@ -1,10 +1,22 @@
 import { Component } from '@angular/core';
+import { ActivatedRoute, Router } from '@angular/router';
 
 @Component({
   selector: 'app-root',
   templateUrl: './app.component.html',
   styleUrls: ['./app.component.scss'],
 })
 export class AppComponent {
-  title = 'ng-material-starter-kit-acms';
+  constructor(
+    public _router: Router,
+    private _activatedRoute: ActivatedRoute
+  ) {}
+
+  redirectTo(route: string) {
+    this._router.navigate([route], { relativeTo: this._activatedRoute });
+  }
+
+  hasRoute(route: string): boolean {
+    return this._router.url.includes(route);
```

Comment:
The same like above,  hacky

Reply:
This is to make sure that you do not see icons to a route on which you already are; there is no hackiness here, we only check the route string and adjust the UI